Dockerfile

```diff
@@ -9,7 +9,8 @@ ARG REACT_SDK_BRANCH="master"
 ARG JS_SDK_REPO="https://github.com/matrix-org/matrix-js-sdk.git"
 ARG JS_SDK_BRANCH="master"
 
-RUN apt-get update && apt-get install -y git dos2unix
+RUN apt-get update && apt-get install -y git dos2unix \
+  build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
Hmm, so why are these needed on Arm? Because of some native node module which not available pre-compiled...?

Reply:
Yes. As per https://www.npmjs.com/package/canvas#compiling:
> If you don't have a supported OS or processor architecture, or you use --build-from-source, the module will be compiled on your system. This requires several dependencies, including Cairo and Pango.

So there's a pre-compiled binary available for amd64, but not for arm.

Reply:
It's also worth noting, in case it's not obvious, that these are only being added to the *builder* image, not the final Docker image.

Reply:
Right, makes sense to me. Could you add a comment in the file to explain it's for compiling canvas on architectures like Arm? Then should be good to merge.